docs/src/content/docs/tr/reference/frontmatter.md

```diff
@@ -0,0 +1,353 @@
+---
+title: Frontmatter Referansı
+description: Starlight'ın desteklediği varsayılan frontmatter alanlarının genel bir bakışı.
+---
+
+Markdown ve MDX sayfalarını, ön yüze verilerini ayarlayarak Starlight'ta özelleştirebilirsiniz. Örneğin, normal bir sayfa, `title` ve `description` alanlarını şu şekilde ayarlayabilir:
+
+```md
+---
+title: Bu proje hakkında
+description: Üzerinde çalıştığım projeyi daha iyi anlayın.
+---
+
+Hakkında sayfasına, Hoş geldiniz!
+
+```
+
+## Frontmatter alanları
+
+### `title` (Gerekli)
```

Comment:
Noticed that “gerekli” is lowercase in the configuration reference. Should it be the same here?

```suggestion
### `title` (gerekli)
```